client/executor/wasmtime/src/instance_wrapper/globals_snapshot.rs

```diff
@@ -17,115 +17,62 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 use super::InstanceWrapper;
-use sc_executor_common::{
-	error::{Error, Result},
-};
+use sc_executor_common::error::{Result, Error};
 use sp_wasm_interface::Value;
-use cranelift_codegen::ir;
-use cranelift_wasm::GlobalIndex;
-use wasmtime_runtime::{ExportGlobal, Export};
+use crate::imports::{into_value, into_wasmtime_val};
+
+struct SavedValue {
+	index: usize,
```

Comment:
It is not immediately obvious what kind of index is that: does it lie in the globals index space? exports? or is it a nth mutable global? I think I see the right answer, but future maintainers could get confused. Hence it would be useful to add some context here.